cmd/sync/runner.go

```diff
@@ -212,8 +233,10 @@ func (r *runner) sync(ctx context.Context, srcRegistry, dstRegistry registry.Int
 		case <-ctx.Done():
 			return microerror.Mask(ctx.Err())
 		case err := <-processGetTagsJobErrCh:
+			errorsTotal.Inc()
```

Comment:
It would be good to grep the code for `microerror.JSON` because there are few places where the error is only printed and execution goes on.